test/testdata/infer/isa_generic.rb.cfg-text.exp

```diff
@@ -268,20 +268,20 @@ bb1[rubyRegionId=0, firstDead=-1]():
 # backedges
 # - bb0(rubyRegionId=0)
 # - bb5(rubyRegionId=1)
-bb2[rubyRegionId=1, firstDead=-1](<self>: T.class_of(Concrete), <C Klass>$10: Runtime object representing type: String, <block-pre-call-temp>$12: Sorbet::Private::Static::Void, <selfRestore>$13: T.class_of(Concrete)):
+bb2[rubyRegionId=1, firstDead=-1](<self>: T.class_of(Concrete), <block-pre-call-temp>$12: Sorbet::Private::Static::Void, <selfRestore>$13: T.class_of(Concrete)):
```

Comment:
Do these come back if there's a reference to `Klass` in the static-init for `Concrete`?

Reply:
Not only a reference, but an assignment, and it has to happen in a loop or block.